packages/cmrjs/index.js

```diff
@@ -150,25 +150,39 @@ async function ingestConcept(type, xml, identifierPath, provider, token) {
  */
 async function deleteConcept(type, identifier, provider, token) {
   const url = `${getUrl('ingest', provider)}${type}/${identifier}`;
+  log.info(`deleteConcept ${url}`);
 
-  const response = await got.delete(url, {
-    headers: {
-      'Echo-Token': token,
-      'Content-type': 'application/echo10+xml'
-    }
-  });
+  let result;
+  try {
+    result = await got.delete(url, {
+      headers: {
+        'Echo-Token': token,
+        'Content-type': 'application/echo10+xml'
+      }
+    });
+  }
+  catch (error) {
+    result = error.response;
+  }
 
   const xmlObject = await new Promise((resolve, reject) => {
-    parseString(response.body, xmlParseOptions, (err, res) => {
+    parseString(result.body, xmlParseOptions, (err, res) => {
       if (err) reject(err);
       resolve(res);
     });
   });
 
-  if (xmlObject.errors) {
-    throw new Error(
-      `Failed to delete, CMR error message: ${JSON.stringify(xmlObject.errors.error)}`
-    );
+  if (xmlObject.errors) log.debug(xmlObject.errors);
+
+  if (result.statusCode !== 200 && result.statusCode !== 404) {
```

Comment:
If we are getting a 404 here and returning success, I think we should at least log that we got a 404 along with the URL that we got the 404 from in case we need to debug that down the road.